mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -439,6 +441,9 @@ private void appendWhere(SelectJoinStep step, ParseLeaderResult parseLeaderResul
           .errors();
 
         recordsLoadingErrors.forEach(error -> {
+          if(error.exception().sqlState().equals(UNIQUE_VIOLATION_SQL_STATE)) {
+            throw new DuplicateEventException("Error when handling duplicate event");
```

Comment:
Can we add some identifiers of duplicate events in message such as jobExecutionId, recordId etc?

Reply:
Unfortunately, it will not succeed. Only record Collection and TenantId are in the method arguments.